Automattic/newspack-plugin
# Newspack Logo component.

### All Submissions:






### Changes proposed in this Pull Request:

Adds a Newspack Logo component (`NewspackLogo`) which can be used wherever we need branding. The component takes optional `width`, `height`, and `className` parameters. 

### How to test the changes in this Pull Request:

1. `npm ci && npm run clean && npm run build:webpack`
2. Navigate to `Newspack->Components Demo`
3. Observe the Newspack logo at the top of the page. 

<img width="1466" alt="Screen Shot 2019-06-26 at 4 55 22 PM" src="https://user-images.githubusercontent.com/1477002/60214435-78443200-9833-11e9-9b26-607829b2e0f2.png">

### Other information:
